js/viz/export.d.ts

```diff
@@ -1,11 +1,24 @@
-import DOMComponent from '../core/dom_component';
-
 import {
-    ExportFormat,
-    HorizontalAlignment,
-    VerticalAlignment,
+  ExportFormat,
+  HorizontalAlignment,
+  VerticalAlignment,
 } from '../common';
 
+import dxChart from './chart';
+import dxPieChart from './pie_chart';
+import dxFunnel from './funnel';
+import dxSankey from './sankey';
+import dxTreeMap from './tree_map';
+import dxVectorMap from './vector_map';
+import dxSparkline from './sparkline';
+import dxBullet from './bullet';
+import dxBarGauge from './bar_gauge';
+
+/**
+ * @docid
+ * @public */
```

Comment:
Format:
```suggestion
/**
 * @docid
 * @public
 */
```

Reply:
About this '@docid' - actually we could remove it. 
It depends whether we want to have a special topic for `VizWidget`, or use a kind of inline description
/cc @vladaskorohodova

Reply:
without @docid validate-declarations does not pass